src/python/pants/engine/internals/graph.py

```diff
@@ -354,7 +354,7 @@ async def resolve_targets(
                     _TargetParametrizations,
                     _TargetParametrizationsRequest(
                         tgt.address.maybe_convert_to_target_generator(),
-                        description_of_origin="TODO(#14468)",
+                        description_of_origin="<infallible>",
```

Comment:
This is when going from `UnexpandedTargets -> Targets`. You should never be manually creating `UnexpandedTargets`...so we can be pretty confident that all the addresses are valid.

(Would it help to comment this? I'm happy to go through every call site and explain why things are infallible.)